.github/workflows/create-kustomize-with-bratiska-cli-inhouse.yml

```diff
@@ -0,0 +1,83 @@
+name: General workflow which creates kustomize using bratiska-cli.
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        description: 'Define the type of machine to run the job on'
+        type: string
+        required: false
+        default: '["self-hosted", "Linux", "X64", "bratislava"]'
+      directory:
+        description: 'A Folder within the repository where things should be build.'
+        default: '/'
+        required: false
+        type: string
+      debug:
+        description: 'Debug flag'
+        default: ''
+        required: false
+        type: string
+      version:
+        description: 'Bratiska-cli version'
+        default: 'stable'
+        required: false
+        type: string
+
+jobs:
+  create-kustomize-with-bratiska-cli:
+    name: Create kustomize with bratiska-cli
+    runs-on: ${{fromJSON(inputs.runs-on)}}
+    defaults:
+      run:
+        working-directory: ${{ inputs.directory }}
+    steps:
+      - name: Checking out
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Pipelines Version
+        run: |
+          echo "Pipelines version: 2.3.0"
+
+      - name: Directory check
+        run: pwd
+
+      - name: Print pipeline summary
+        run: |
+          echo "### Bratiska-cli is building:" >> $GITHUB_STEP_SUMMARY
+          echo ":arrow_right: Project folder: **${{ inputs.directory }}**" >> $GITHUB_STEP_SUMMARY
+
+      - name: Install Prerequisites -  Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Installing Bratiska-cli version
+        run: yarn global add bratislava/bratiska-cli#${{ inputs.version }}
+
+      - name: Running git fetch for Bratiska-cli
+        run: git fetch
+
+      - name: Print Bratiska-cli version
+        run: bratiska-cli --version
+
+      - name: Creating kustomize file with Bratiska-cli
+        if: ${{ steps.secret-check.outputs.available != 'true' }}
+        run: bratiska-cli build_kustomize --env=staging ${{ inputs.flag }} ${{ inputs.debug }} --no_image_repo_check --image=image_url_placeholder
+        continue-on-error: false
```

Comment:
1. Same question here. Do you think we could switch to something more standard, from community, so we don't have to maintain as much? We wouldn't have to have that many steps and for example, not install node for all of the actions, etc.    
Something like this https://github.com/marketplace/actions/kustomize-github-action. It has 45 stars and seems to be used by multiple people.

2. Moreover, with all of these flags this is basically just alias for https://github.com/bratislava/bratiska-cli/blob/master/src/commands.ts#L395, so maybe it would be better to just replace it with that command.   
If you wouldn't like to use the above mentioned GitHub action. 

3. And lastly, we is it frozen here on `--env=staging`?

Reply:
ok, but I'm still wondering why the all the steps have a hard-coded `--env=staging`?

Reply:
Ok, sure. I would in the future prefer if this should also be used as a test, that it would be fully autonomous and wouldn't relly on staging. But for now it's good. Thanks for the explanation 👍